invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/RecordResourceActions.js

```diff
@@ -28,6 +29,22 @@ export class RecordResourceActions extends Component {
   onModalTriggerClick = (e, { payloadSchema, dataName, dataActionKey }) => {
     const { resource } = this.props;
 
+    if (dataActionKey === "compare") {
+      this.setState({
+        modalOpen: true,
+        modalHeader: i18next.t("Compare revisions"),
+        modalProps: {
```

Comment:
Is this `modalProps` used? or just a leftover

Reply:
I updated the implementation, it was not used indeed :)